r/vignettes/fs.Rmd

```diff
@@ -144,3 +147,28 @@ s3://minioadmin:minioadmin@?scheme=http&endpoint_override=localhost%3A9000
 
 Among other applications, this can be useful for testing out code locally before
 running on a remote S3 bucket.
+
+## Disabling the use of environment variables
+
+As mentioned above, it is possible to make use of environment variables to 
+configure access.  However, if you wish to pass in connection details via a URI
+or alternative methods but also have existing AWS environment variables defined,
+these may interfere with your session.  You can unset these environment 
+variables using `Sys.unsetenv()`, for example:
+
+```{r, eval = FALSE}
+Sys.unsetenv("AWS_DEFAULT_REGION")
+Sys.unsetenv("AWS_S3_ENDPOINT")
```

Comment:
Should this instead be a snippet which contains instructions for unsetting env vars and then resetting at end of session?